frontend/src/components/EvaluationResponses.js

```diff
@@ -55,171 +78,317 @@ const EvaluationResponses = ({ crn, info }) => {
   // Sort by original order or length?
   const [sort_order, setSortOrder] = useState('original');
 
-  const sortByLength = useCallback((responses) => {
-    for (const key in responses) {
-      responses[key].sort(function (a, b) {
-        return b.length - a.length;
+  // Hooks for filtering evaluations in the searchbar
+  const [data, setData] = useState([]);
+  const [dataDefault, setDataDefault] = useState([]);
+  const [dataDefault2, setDataDefault2] = useState([]);
+  const [keyword, setKeyword] = useState('');
+  const [dataChange, setDataChange] = useState([]);
+  const [curPanel, setCurPanel] = useState('');
+
+  // Used to sort frequency of adjectives in each evaluation
+  const sortByFrequency = (array) => {
+    const frequency = {};
+    array.forEach((value) => {
+      frequency[value] = 0;
+    });
+    const uniques = array.filter((value) => {
+      return ++frequency[value] === 1;
+    });
+    return uniques.sort((a, b) => {
+      return frequency[b] - frequency[a];
+    });
+  };
+
+  const summarize_comments = useRef(null);
```

Comment:
Use camelCase to match standard in this file

Reply:
Will do.